modules/eks/versions.tf

```
@@ -0,0 +1,20 @@
+terraform {
+  required_version = ">= 0.13.1"
```

```suggestion
  required_version = ">= 0.15"
```

0.15 is safer in this case because this is not going to be used by customers running ancient versions of Terraform.

`terraform-aws-modules` are being used by very many customers, so we use `~> 0.13`.